Xamarin.Forms.Core/ScrollOrientation.cs

```diff
@@ -4,6 +4,7 @@ public enum ScrollOrientation
 	{
 		Vertical,
 		Horizontal,
-		Both
+		Both,
+		Neither
```

Comment:
I'd go for `None`

Reply:
/cc @samhouts

Reply:
Hm, I'm not absolutely sure, but I think that "Neither" is more accurate antonym for "Both".
https://english.stackexchange.com/questions/50693/what-is-the-difference-between-none-of-and-neither-of

Reply:
I like `Neither`. We set the stage for the silly naming with `Both`. For better or worse, it now reads like a question in English. "Which orientation should this scroll: Vertical, Horizontal, Both, or Neither?" 

If instead of `Both`, we had used `Bidirectional` or something, then I would have fully supported `None`.

Reply:
@StephaneDelcroix do you agree on `Neither`?